alaoweb/2020.alaoweb.org
# base64 encode awardees data file values

* partially addresses #274 
* `_awards.yml` replaced with `awards-base64.json`
* the json file uses encoded strings for the "value" part of the json "key":"value" pairs
* adds a custom ruby filter to decode values at build time
* the awards.html page will still accept un-encoded values when `{% use64 = false %}`
* how to create the base64-encoded json file will be a separate PR
* n.b.: base64 encoding obfuscates the info; a sufficiently motivated person can decode it easily, but not accidentally

## to test: 
* check out branch
* in `_config.yml` set `showAwardWinners: true`
* build and serve
* go to: http://localhost:4000/awards/
 * awards should display normally
* in `awards.html`, set `{% use64 = false %}` and reload the page: should be gibberish values
